### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- `CHANGELOG.md` updated
- `alpha` added to save/load system
- docstrings updated
- autopep8
- `AUTHORS.md` updated
- `overall_test.py` updated

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri Please review this PR 🥇 
If everything is okay, **Squash/Merge** it after approval.

File: AUTHORS.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
 ----------
-- Sepand Haghighi - Sharif University Of Technology/Moduland Co - [@sepandhaghighi](http://github.com/sepandhaghighi)
-- Sadra Sabouri - Sharif University Of Technology - [@sadrasabouri](https://github.com/sadrasabouri)
+- Sepand Haghighi - Sharif University of Technology - [@sepandhaghighi](http://github.com/sepandhaghighi)
+- Sadra Sabouri - Sharif University of Technology - [@sadrasabouri](https://github.com/sadrasabouri)
 
 # Other Contributors #
-- Mohammad Hosein Moslemi - Sharif University Of Technology - [@mhmoslemi2338](https://github.com/mhmoslemi2338)
+- Mohammad Hosein Moslemi - [@mhmoslemi2338](https://github.com/mhmoslemi2338)
```
Comment (sadrasabouri) [inline AUTHORS.md:8]: Why did you delete this?
He is from Sharif University, no problem.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]

File: AUTHORS.md (hunk 1 above)
Comment (sepandhaghighi) [reply AUTHORS.md:8]: It's all about the format, nothing else

**Review by sadrasabouri — APPROVED** [review]
